etc/NetworkCommunication.conf

```diff
@@ -0,0 +1,6 @@
+[NetworkCommunication]
+	CC_ROBOT_IP_BASE_0 = 172
+	CC_ROBOT_IP_BASE_1 = 16
+	CC_ROBOT_IP_BASE_2 = 40
+	CC_ROBOT_IP_BASE_3 = 10
+[!NetworkCommunication]
```

Comment:
Probably required for changes in msldriver, good additions to the color calibrator